conf/officialsAndNominees.routes

```diff
@@ -5,36 +5,50 @@ GET        /:index/authorised-official-name
 POST       /:index/authorised-official-name                                     controllers.authorisedOfficials.AuthorisedOfficialsNameController.onSubmit(mode: Mode = NormalMode, index: Index)
 GET        /:index/authorised-official-name/change                              controllers.authorisedOfficials.AuthorisedOfficialsNameController.onPageLoad(mode: Mode = CheckMode, index: Index)
 POST       /:index/authorised-official-name/change                              controllers.authorisedOfficials.AuthorisedOfficialsNameController.onSubmit(mode: Mode = CheckMode, index: Index)
+GET        /:index/authorised-official-name/playback-change                              controllers.authorisedOfficials.AuthorisedOfficialsNameController.onPageLoad(mode: Mode = PlaybackMode, index: Index)
```

Comment:
please keep the consistent alignments , for all the changes in file

Reply:
all fixed now